easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -230,8 +231,13 @@ def __init__(self, *args, **kwargs):
         elif self.cfg.get('use_pip', False):
             self.install_cmd = PIP_INSTALL_CMD
 
-            # don't auto-install dependencies
-            self.cfg.update('installopts', '--no-deps')
+            # don't auto-install dependencies with pip unless use_pip_for_dependencies=True
+            # the default is use_pip_for_dependencies=False
+            if self.cfg.get('use_pip_for_dependencies') == False:
+                self.log.info("Using pip with --no-deps option")
+                self.cfg.update('installopts', '--no-deps')
+            else:
+                self.log.info("Using pip to also install the dependencies")
```

Comment:
Checking `== False` is a bit weird @pescobar, since it's a boolean... Should be:

```python
# don't auto-install dependencies with pip, unless use_pip_for_deps is enabled
if self.cfg.get('use_pip_for_deps'):
    self.log.info("Using pip to also install the dependencies")
else:
    self.log.info("Using pip with --no-deps option")
    self.cfg.update('installopts', '--no-deps')